adoc/TRD-fep-vm-sles-s390x-getting-started.adoc

```diff
@@ -20,32 +20,36 @@
 
-SUSE Linux Enterprise Server is trusted by enterprises around the world for their mission-critical IT deployments. A multimodal operating system that paves the way for IT transformation in the software-defined era. The modern and modular OS helps simplify multimodal IT, makes traditional IT infrastructure efficient and provides an engaging platform for developers. As a result, organiztions can easily deploy and transition business-critical workloads across on-premise and public cloud environments.
+SUSE Linux Enterprise Server is trusted by enterprises around the world for their mission-critical IT deployments. A multimodal operating system that paves the way for IT transformation in the software-defined era. The modern and modular OS helps simplify multimodal IT, makes traditional IT infrastructure efficient and provides an engaging platform for developers. As a result, organiztions can easily deploy and transition business-critical workloads across on-premises and public cloud environments.
 
 Fujitsu has been developing enterprise, mission-critical database systems for over 35 years.  Since 2004, Fujitsu has provided development, support, training, and consulting services to businesses worldwide for PostgreSQL, one of the world's most advanced, open source database systems.  Fujitsu Enterprise Postgres (FEP) remains compatible with open source PostgreSQL, while adding enhanced features, such as in-memory columnar indexing, data masking, transparent data encryption, and dedicated audit logging.
 
-IBM Z and LinuxONE systems are enterprise-class, purpose-built data serving platform for mission critical Linux business workloads that mandate the highest levels of security, scalability, performance and resilience. The platform supports traditional n-tier Linux applications and databases, as well as cloud-ready containerized workloads and microservices, delivering a true secure hybrid cloud platform for all organizations.
+IBM Z and LinuxONE systems are an enterprise-class, purpose-built, data serving platform for business-critical Linux workloads that mandate the highest levels of security, scalability, performance, and resilience. The platform supports traditional n-tier Linux applications and databases, as well as cloud-ready containerized workloads and microservices, delivering a true, secure, hybrid-cloud platform for organizations of all sizes.
 
 
 == High-level overview
 
-This document will guide a administrator through the steps to deploy Fujitsu Enterprise Postgres (FEP) 12 on a SUSE Linux Enterprise Server (SLES) 12 SP5. SLES 12 SP5 is the version that supported by FEP12. The end result of these steps is FEP running on SLES with a test database.
+This document guides a administrator through installation of Fujitsu Enterprise Postgres (FEP) 12 on a SUSE Linux Enterprise Server (SLES) 12 SP5, leveraging automation to reduce the time required to deploy both SLES and FEP.
```

Comment:
Please fix "This document guides a administrator" with "This document guides an administrator".

Reply:
Done and pushed